server/schedule/operator_controller.go

```diff
@@ -94,6 +94,7 @@ func (oc *OperatorController) AddOperator(ops ...*Operator) bool {
 	return true
 }
 
+// checkAddOperator check if the operator can be added.
```

Comment:
checks

Reply:
And explain when an operator cannot be added.